### DISCUSSION TITLE
FanControl does not start minimized.

### DISCUSSION THREAD

**Discussion by stani63** [Rem0o/FanControl.Releases#2486] [Q&A]
Of course "Start Minimized" is set in its settings. 
Of course, in the Windows "Startup" folder, "Run: in a minimized window" is also set. 
I also tried to add "StartMinimized": true, in the FanControl - Configurations folder in the "userConfig.json" file. It didn't help either. 
I also tried manually running "Updater.exe" in the FanControl folder. 

None of this helped, after starting Windows, the entire FanControl window always appears, which I always have to manually close. 

But it is very interesting that there is no problem on another, completely identical PC with the same W11.

**Comment by Rem0o** [comment]
The start minimized option is actually in the CACHE file, not the json config files.

Do you get any pop-up dialog when the window opens by itself at startup?

**Reply by stani63** [reply]
Hi, none "any pop-up dialog"...

**Comment by stani63** [comment]
Solved. Where it didn't work was the Portable version.
